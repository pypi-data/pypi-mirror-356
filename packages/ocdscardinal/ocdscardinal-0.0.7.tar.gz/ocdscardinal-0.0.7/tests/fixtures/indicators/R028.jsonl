{"ocid":"F","bids":{"details":[{"status":"valid","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"V"}]},{"status":"disqualified","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"D"}]}]}}
{"ocid":"missing tenderers","bids":{"details":[{"status":"valid","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"V"}]},{"status":"disqualified","value":{"amount":1,"currency":"USD"}},{"status":"disqualified","value":{"amount":1,"currency":"USD"}}]}}
{"ocid":"same tenderers","bids":{"details":[{"status":"valid","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"T"}]},{"status":"disqualified","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"T"}]}]}}
{"ocid":"shuffled tenderers","bids":{"details":[{"status":"valid","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"A"},{"id":"Z"}]},{"status":"disqualified","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"Z"},{"id":"A"}]}]}}
{"ocid":"different decimals","bids":{"details":[{"status":"valid","value":{"amount":1.0,"currency":"USD"},"tenderers":[{"id":"V"}]},{"status":"disqualified","value":{"amount":1.1,"currency":"USD"},"tenderers":[{"id":"D"}]}]}}
{"ocid":"different currencies","bids":{"details":[{"status":"valid","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"V"}]},{"status":"disqualified","value":{"amount":1,"currency":"CAD"},"tenderers":[{"id":"D"}]}]}}
{"ocid":"cancelled tender","bids":{"details":[{"status":"valid","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"V"}]},{"status":"disqualified","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"D"}]}]},"tender":{"status":"cancelled"}}
{"ocid":"no price comparison","bids":{"details":[{"status":"valid","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"V"}]},{"status":"disqualified","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"D"}]}]},"tender":{"procurementMethodDetails":"NPC"}}
{"ocid":"exclude procurement method details","bids":{"details":[{"status":"valid","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"V"}]},{"status":"disqualified","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"D"}]}]},"tender":{"procurementMethodDetails":"EXC"}}
