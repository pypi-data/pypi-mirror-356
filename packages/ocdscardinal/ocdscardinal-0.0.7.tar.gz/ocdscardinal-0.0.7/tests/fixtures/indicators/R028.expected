{"OCID":{"F":{"R028":1.0}},"Tenderer":{"D":{"R028":1.0},"V":{"R028":1.0}}}
